src/app/containers/ConsentBanner/Canonical/logic/index.test.js

```diff
@@ -0,0 +1,236 @@
+/* eslint-disable global-require */
+const PRIVACY_COOKIE = 'ckns_privacy';
+const EXPLICIT_COOKIE = 'ckns_explicit';
+const POLICY_COOKIE = 'ckns_policy';
+
+let Cookie;
+let setCookieOvenMock;
+const setShowPrivacyBannerMock = jest.fn();
+const setShowCookieBannerMock = jest.fn();
+
+const setCookieGetMock = ({ privacy = '1', explict = '1', policy = '111' }) => {
+  Cookie.get.mockImplementation(cookie => {
+    if (cookie === PRIVACY_COOKIE) {
+      return privacy;
+    }
+
+    if (cookie === EXPLICIT_COOKIE) {
+      return explict;
+    }
+
+    return policy;
+  });
+};
+
+const getCanonicalLogic = ({ logger } = {}) => {
+  const canonicalLogic = require('./index').default;
+  return canonicalLogic({
```

Comment:
The references to 'canonical logic' in this file are quite abstract and make it difficult to read. Instead, could you call this `cookies` as you have done in the readme?

```suggestion
  return cookies({
```

Likewise everywhere you've used `getCanonicalLogic` to replace it with `consentBannerUtilities`.

Reply:
Agreed on slack naming, done :)